plugins/entity-feedback/src/components/FeedbackResponseDialog/FeedbackResponseDialog.tsx

```diff
@@ -102,13 +136,29 @@ export const FeedbackResponseDialog = (props: FeedbackResponseDialogProps) => {
     }
   }, [comments, consent, entity, feedbackApi, onClose, responseSelections]);
 
+  const selectMandatedBox = (res: boolean) => {
+    const newResponseSelections = { ...responseSelections };
+    requireComments.forEach(id => newResponseSelections[id] === res);
```

Comment:
Hi @kuangp the idea here is to link the comment boxes to specific check boxes. 
- if a user checks a box where mustComment is true, comments must be submitted.
- if a user submits comments when no box is checked, select the tied response. 
This was requested by our design team, but if this is not something that backstage wants to support, please advise on what the behavior should be.

Reply:
I think the behavior described in your second point is potentially unintuitive, perhaps you can bring this back to your design team for discussion:

Consider the following example - your feedback dialog has the following options:
  - This entity is deprecated (mustComment)
  - This entity is incorrect (mustComment)
  - Other (mustComment)

Now if the user enters some comments this will auto select all options. However, we don't know which option the user intends to comment on and whether every option is actually applicable. The user may have only wanted to provide generic comments but not intend to state that the entity is either deprecated nor incorrect. A confused user would have to go back and unselect those options wondering why they got auto selected for them.